dev/com.ibm.ws.os.packaging_fat/fat/src/com/ibm/ws/os/packaging/fat/InstallRhelTest.java

```diff
@@ -62,24 +64,16 @@ public static void cleanup() throws Exception {
     @Test
     public void testVerifyRpmInstall() throws Exception {
 
-        if (openLibExists) {
-            String METHOD_NAME = "testVerifyRpmInstall";
-            entering(c, METHOD_NAME);
-
-            String[] param1s = { "-qi", "openliberty" };
-            ProgramOutput po = runCommand(METHOD_NAME, "rpm", param1s);
-            assertEquals("Expected exit code", 0, po.getReturnCode());
-            exiting(c, METHOD_NAME);
-        } else {
-            logger.info("OpenLiberty did not install successfully");
-        }
-    }
+        String METHOD_NAME = "testVerifyRpmInstall";
+        entering(c, METHOD_NAME);
+        assumeTrue(openLibExists);
```

Comment:
I would recommend we pick one of these, either assume or assert. I don't think there's any way line 70 executes if `openLibExists` is false.

Reply:
changed to use assert.